tests/safety/test_hyundai.py

```diff
@@ -280,7 +319,7 @@ def test_buttons(self):
   def _pcm_status_msg(self, enable):
     raise NotImplementedError
 
-  def _button_msg(self, buttons):
+  def _button_msg(self, buttons, main_button=0):
```

Comment:
does this actually need to be redefined? should be the same msg right?

Reply:
Yes, for linter reasons